src/Atoms/MainBtn/MainBtnStyled.js

```diff
@@ -0,0 +1,47 @@
+import styled from 'styled-components';
+import COLORS from '../../Styles/varsStyledComponents';
+import FONTS from '../../Styles/fontsStyledComponents';
+
+export const Wrapper = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+`;
+
+const MainBtnStyled = styled.button`
+  display: block;
+  width: 343px;
+  height: 50px;
+  color: ${(props) =>
+    !props.primary
```

Comment:
it smells a little bit like something for Themes in Styled components? But not in that issue, just as TODO - think in a team about themes

Reply:
TODO.